### PR TITLE
Created ECE385 with ECE391 and ECE210 with CS225 pages

### PR DESCRIPTION
**Pull request by upai02**

### CODE REVIEW

File: docs/Course Wiki/Taking X with Y/ECE385 with ECE391.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# ECE385 with ECE391
+
+## Relevant Course Reviews
+- [ECE385](../ECE%20Course%20Offerings/ECE385.md)
+- [ECE391](../ECE%20Course%20Offerings/ECE391.md)
+
+## Who should take them?
+CompE majors are required to take both courses, while EEs only need to take [ECE385](../ECE%20Course%20Offerings/ECE385.md) but can take [ECE391](../ECE%20Course%20Offerings/ECE391.md) for their hardware lab requirement. Both courses do not have much overlap in terms of content, as ECE385 focuses more on hardware design whereas ECE391 focuses on low-level software design. There are some shared topics about device interfacing which are covered from different perspectives in each class, however the courses cover mostly topics that are at best tangentially related. Taking these courses together builds a great understanding of the hardware-software interface; ECE391 and can help with ECE385 in that by understanding how the software uses the hardware will make it easier to design easy to program software-hardware interfaces (e.g. for the ECE385 final project).  Having said this, both courses work very well when taken independently, and it is fairly common to take ECE385 before ECE391.
+
+## Workload:
```
Comment (rzig) [inline docs/Course Wiki/Taking X with Y/ECE385 with ECE391.md]: remove : after header

File: docs/Course Wiki/Taking X with Y/ECE385 with ECE391.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+Both courses are very time-intensive. [ECE391](../ECE%20Course%20Offerings/ECE391.md) has 3 MPs throughout the semester, but they are all fairly large projects - even the fastest students often spend a minimum of 20 hours a week on these MPs. [ECE385](../ECE%20Course%20Offerings/ECE385.md), on the other hand, is typically around 10 hours of work per week, since each week you have to design and build/implement the circuit for the next lab and finish your lab report for the previous week's lab; both these classes can take significantly more time if you are not good at debugging your MPs & digital circuits. The main thing to know, though, is that you'll be spending at least half your time on labs & MPs if you take these classes concurrently.
```
Comment (rzig) [inline docs/Course Wiki/Taking X with Y/ECE385 with ECE391.md]: 20 hours a week -> 20 hours per week

File: docs/Course Wiki/Taking X with Y/ECE385 with ECE391.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+Additionally, both courses have final projects with varying levels of open-endedness. If you want to put extra effort and time into these, you'll probably have to choose whether to focus your effort on ECE385's final project (which can be whatever digital circuit you want, subject to TA approval) and ECE391's MP3, which is a group MP in which students get to design their own UNIX-like operating system.
+
+## Recommendation:
```
Comment (rzig) [inline docs/Course Wiki/Taking X with Y/ECE385 with ECE391.md]: remove :

File: docs/Course Wiki/Taking X with Y/ECE385 with ECE391.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+If you are still thinking twice about taking these courses together, a good alternative would be to take ECE391 and a technical elective (there are a lot of CompE tech electives you can take after ECE120, ECE220, and [CS225](../CS%20Course%20Offerings/CS225.md) which aren't huge time sinks).  Why not take ECE385 first, you might ask?  Well, there are many classes that depend on ECE391, whereas almost none require ECE385.
```
Comment (rzig) [inline docs/Course Wiki/Taking X with Y/ECE385 with ECE391.md:18]: link 120 and 220

**Review by rzig — DISMISSED** [review]

**Review by lnguy7 — APPROVED** [review]

File: docs/Course Wiki/Taking X with Y/ECE210 with CS225.md [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+
+Meanwhile, [CS225](../CS%20Course%20Offerings/CS225.md) has 6 MPs occuring biweekly, a weekly lab, 6 exams, and a final. MPs are usually divided into 2 parts, where the first part can be completed a week early for extra credit, while the whole MP is due all at once after the 2 weeks are up. Weekly labs can usually be done during the lab sections and are similar in length to [ECE220](../ECE%20Course%20Offerings/ECE220.md) MPs, while the MPs are usually much larger in scale. One can expect to spend around 10-15 hours per week on these assignments. There is also an optional group-based final project for extra credit, which some students may choose to pursue for various reasons.
+
+## Recommendation:
+Yes, taking these two courses together is very much doable. Since these two courses are not related, they can be and commonly are taken together, especially by CompE students who took [ECE110](../ECE%20Course%20Offerings/ECE110.md) and [ECE220](../ECE%20Course%20Offerings/ECE220.md) together.
```
Comment (curtstutsman) [inline docs/Course Wiki/Taking X with Y/ECE210 with CS225.md:16]: Maybe add something how the main focus of 210 is the exams, whereas the exams in 225 are much easier and more emphasis is put on the MPs

**Pull request merged**